packages/common/container-definitions/package.json

```diff
@@ -47,7 +47,7 @@
   "devDependencies": {
     "@fluidframework/build-common": "^0.23.0",
     "@fluidframework/build-tools": "^0.2.71273",
-    "@fluidframework/container-definitions-previous": "npm:@fluidframework/container-definitions@0.48.1000",
+    "@fluidframework/container-definitions-previous": "npm:@fluidframework/container-definitions@^1.0.0",
```

Comment:
no caret here -- you want a specific version (I think)

Reply:
You are right. It needs to be specific. I'll follow up with another PR as it may be more then these 3 packages.